pipeline/terraform/stack/elastic_pipeline.tf

```diff
@@ -83,6 +83,7 @@ resource "aws_secretsmanager_secret" "es_username_catalogue" {
   for_each = toset([
     "snapshot_generator",
     "catalogue_api",
+    "concepts_api",
```

Comment:
terraform apply was failing in create_pipeline_storage_users because these were missing